tests/test.sh

```diff
@@ -1,17 +1,15 @@
 #!/bin/bash
 
 trap "exit 130" INT
-cleanup() { rm -f test.ini good.ini example.ini; exit; }
+cleanup() { rm -f test.ini good.ini tests/example.ini; exit; }
 trap cleanup EXIT
 
-export PATH=..:$PATH
-
```

Comment:
If the current or parent dir isn't in the $PATH, we're not sure the correct crudini will be executed